src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -259,4 +258,24 @@ public void onFailure(Throwable t) {
         executor.awaitTermination(1, TimeUnit.MINUTES);
     }
 
+    /**
+     * Returns an {@link ExecutorService} to be used as a parameter in other methods.
+     * It calls {@code MoreExecutors#newDirectExecutorService} or falls back to {@code MoreExecutors#sameThreadExecutor}
+     * for compatibility with older (< 18.0) versions of guava.
+     */
+    private static ExecutorService newExecutorService() {
+        try {
+            try {
+                Method method = MoreExecutors.class.getMethod("newDirectExecutorService");
```

Comment:
can you invert this please as in [`c5ecc11` (#114)](https://github.com/jenkinsci/workflow-support-plugin/pull/114/commits/c5ecc11733196e2dc054a899e846be31e8a0b1e4) (was requested until core PR is merged)